src/main/java/at/hannibal2/skyhanni/features/misc/GeorgeDisplay.kt

```diff
@@ -0,0 +1,132 @@
+
+    @SubscribeEvent
+    fun onInventoryOpen(event: InventoryOpenEvent) {
+        if (!isEnabled()) return
+        if (!offerPetsChestPattern.matches(event.inventoryName)) return
+        val stack = event.inventoryItems[41] ?: return //the slot #41 = spawn egg with the tooltip
+        if (!increaseCapItemPattern.matches(stack.cleanName())) return
+        display = drawDisplay(stack.getLore())
+    }
+
+    private fun drawDisplay(lore: List<String>) = buildList<Renderable> {
+        this.add(
+            Renderable.string(
+                "§d§lTaming 60 Cost: §r§d(${
+                    if (config.otherRarities) "cheapest" else "exact"
+                } rarity)"
+            )
+        )
+        var totalCost = 0.0
+        for (line in lore)
+            neededPetPattern.matchMatcher(line) {
+                val origTierString = group("tier")
+                val origPetString = group("pet")
+                val pet = origPetString.convertToInternalNameString().removePrefix("FROST_")
+                val originalTier = LorenzRarity.getByName(origTierString.uppercase().replace(" ", "_"))?.id ?: 0
+                val (cheapestTier, petPrice) = findCheapestTier(pet, originalTier)
+                val displayPetString = "${LorenzRarity.getById(cheapestTier)?.formattedName} $origPetString"
+                if (petPrice != -1.0) {
+                    totalCost += petPrice
+                    this@buildList.add(
+                        Renderable.clickAndHover(
+                            text = " §7- $displayPetString§7: §6${petPrice.addSeparators()} coins",
+                            tips = listOf(
+                                "§aClick to run §e/ahs ] $origPetString §ato find it on the Auction House.",
+                                "§aNotes: §eSet the rarity filter yourself. §cBooster Cookie required!"
+                            ),
+                            onClick = { LorenzUtils.sendCommandToServer("ahs ] $origPetString") }
+                        ))
+                } else {
+                    val typeOfWiki = if (useFandomWiki) "Fandom" else "Official Hypixel"
+                    this@buildList.add(
+                        Renderable.clickAndHover(
+                            text = " §7- $displayPetString§7: §eNot on AH; view its $typeOfWiki article here.",
+                            tips = listOf("§eClick to open the $typeOfWiki Wiki article for the $displayPetString §eto view how to obtain it."),
+                            onClick = {
+                                val urlCompliantPet = displayPetString.removeColor().replace(" ", "%20")
+                                if (useFandomWiki) OSUtils.openBrowser("https://hypixel-skyblock.fandom.com/wiki/Special:Search?query=$urlCompliantPet&scope=internal")
+                                else OSUtils.openBrowser("https://wiki.hypixel.net/index.php?search=$urlCompliantPet")
+                            }
+                        ))
+                }
```

Comment:
How often do pets not have an ah price, this seems like they have an issue connecting to neu lowestbin api and that issue should be adressed properly instead of adding a weird work around

Reply:
I think this is for the soulbounded pets